app/src/main/java/com/shobhith/atvproductsapp/home/domain/use_case/GetProductCategories.kt

```diff
@@ -0,0 +1,8 @@
+package com.shobhith.atvproductsapp.home.domain.use_case
+
+import com.shobhith.atvproductsapp.home.domain.repository.ProductRepository
+
+class GetProductCategories(private val repository: ProductRepository) {
+    operator fun invoke() =
+        repository.getProductCategories()
+}
```

Comment:
Just one suggestion always try to add one blank line at the end of every file. Now it's not an issue. But lint will throw an warning.